wording-guidelines.md

```diff
@@ -1,41 +1,61 @@
 # Wording Guidelines
 
-Jens Maurer, 2025-02-27
+Jens Maurer et al., 2025-02-28
 
 This document offers a collection of guidelines for writing wording
-targeted at integration into the ISO C++ Working Draft,
-with a focus on core language wording (clauses 2-15).
+targeted at integration into the [ISO C++ Working Draft][draft],
+with a focus on core language wording
+(clauses [[intro.refs]](https://eel.is/c++draft/intro.refs) through
```

Comment:
eel.is is unofficial; I think I don't want to refer to it in these guidelines at all.

(I'll pick the parts I like for a future update of this document.)

Reply:
Perhaps add a disclaimer that the eel.is links are not an endorsement at the beginning? They can still be pretty helpful, and we don't have any way of stable-linking to content within the latest draft.

Reply:
I feel we do want less linking and more work for the authors to actually read the Working Draft, as much as possible.